dist/lrm-mapzen.js

```diff
@@ -1400,6 +1402,14 @@ if (typeof module !== undefined) module.exports = polyline;
 			if (route.subRoutes) {
 				for(var i = 0; i < route.subRoutes.length; i++) {
 					if(!route.subRoutes[i].styles) route.subRoutes[i].styles = this.options.styles;
+					if (route.subRoutes[i].travel_type=='foot') {
```

Comment:
since there is way to pass styles of line in lrm , setting manual style inside of lrm-mapzen wouldn't be important.